MyHeartCounts/Utils/ViewFormMatchingBackgroundModifier.swift

```diff
@@ -0,0 +1,34 @@
+//
+// This source file is part of the My Heart Counts iOS application based on the Stanford Spezi Template Application project
+//
+// SPDX-FileCopyrightText: 2025 Stanford University
+//
+// SPDX-License-Identifier: MIT
+//
+
+import SwiftUI
+import class UIKit.UIColor
+
+
+private struct ViewFormMatchingBackgroundModifier: ViewModifier {
+    @Environment(\.colorScheme)
+    private var colorScheme
+    
+    func body(content: Content) -> some View {
```

Comment:
Agree; might be nice to move this to SpeziViews at some point, we have this across a few applications ...